internal/schemautil/mutations.go

```diff
@@ -30,6 +30,22 @@ func normalizeIpFilter(old, new map[string]interface{}) {
 		return
 	}
 
+	fieldToWrite := "ip_filter"
+
+	if len(oldIpFilters)*len(newIpFilters) == 0 {
```

Comment:
So if newIpFilters is empty and oldIpFilters is not should write to ip_filter_object?
Is this correct?

Reply:
yes